flyteadmin/pkg/repositories/transformers/execution.go

```diff
@@ -91,11 +93,24 @@ func CreateExecutionModel(input CreateExecutionModelInput) (*models.Execution, e
 	if input.Phase == core.WorkflowExecution_RUNNING {
 		closure.StartedAt = createdAt
 	}
+	if input.Error != nil {
+		closure.Phase = core.WorkflowExecution_FAILED
+		execError := &core.ExecutionError{
+			Code:    "Unknown",
+			Message: input.Error.Error(),
+			Kind:    core.ExecutionError_UNKNOWN,
+		}
```

Comment:
If we choose to define the error as having an `ExecutionError` s there a reason we can't maintain it through to here?

Reply:
I refactored code to pass ExecutionError explicitly. Not sure if this is what you meant.